src/test/ui/ufcs/ufcs-qpath-missing-params.stderr

```diff
@@ -1,8 +1,16 @@
-error[E0107]: wrong number of type arguments: expected 1, found 0
+error[E0107]: wrong number of type arguments for `IntoCow`: expected 1, found 0
   --> $DIR/ufcs-qpath-missing-params.rs:14:5
    |
+LL | pub trait IntoCow<'a, B: ?Sized> where B: ToOwned {
+   | ------------------------------------------------- defined here, with 1 type parameter: `B`
+...
 LL |     <String as IntoCow>::into_cow("foo".to_string());
-   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected 1 type argument
+   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ found no type argument
+   |
+help: use angle brackets to add missing type argument:
+   |
+LL |     <String as IntoCow>::into_cow::<B>("foo".to_string());
+   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Ideally we would use `shrink_to_hi` and `span_suggestion_verbose` so the output is easier for people to tell what was added:

```suggestion
LL |     <String as IntoCow>::into_cow::<B>("foo".to_string());
   |                                  ^^^^^
```

Reply:
This, plus the suggestion is actually wrong - it should suggest `IntoCow<B>` instead of `into_cow::<B>`.

I've managed to get it to say:
```
error[E0107]: wrong number of type arguments for `IntoCow`: expected 1, found 0
  --> main.rs:27:16
   |
27 |     <String as IntoCow>::into_cow("foo".to_string());
   |                ^^^^^^^
   |                |
   |                expected 1 type argument
   |
note: trait defined here, with 1 type parameter: `B`
  --> main.rs:16:11
   |
16 | pub trait IntoCow<'a, B: ?Sized> where B: ToOwned {
   |           ^^^^^^^     -
help: use angle brackets to add missing type argument:
   |
27 |     <String as IntoCow<B>>::into_cow("foo".to_string());
   |                       ^^^
```

Reply:
fixed 🎉